diff --git a/RecoTauTag/RecoTau/plugins/DeepTauId.cc b/RecoTauTag/RecoTau/plugins/DeepTauId.cc
index 1c2534f3784..f34623b990a 100644
--- a/RecoTauTag/RecoTau/plugins/DeepTauId.cc
+++ b/RecoTauTag/RecoTau/plugins/DeepTauId.cc
@@ -1160,7 +1160,7 @@ public:
     desc.add<bool>("disable_dxy_pca", false);
     desc.add<bool>("disable_hcalFraction_workaround", false);
     desc.add<bool>("disable_CellIndex_workaround", false);
-    desc.add<bool>("save_inputs", false);
+    desc.add<bool>("save_inputs", true);
     desc.add<bool>("is_online", false);
 
     desc.add<std::vector<std::string>>("VSeWP", {"-1."});
